apps/account-api/src/services/accounts.service.ts

```diff
@@ -70,7 +71,7 @@ export class AccountsService {
 
   // eslint-disable-next-line class-methods-use-this
   async signInWithFrequency(request: WalletLoginRequestDto): Promise<WalletLoginResponseDto> {
-    const api = await this.blockchainService.getApi();
+    const api = (await this.blockchainService.getApi()) as ApiPromise;
```

Comment:
question: why do we need this type casting?

Reply:
Because `getApi()` can return one of 2 things, depending on whether you pass in an optional block hash argument: either the base `ApiPromise` -or- the result of `api.at(blockHash)`, which is not an ApiPromise, but is instead `ApiDecoration<'promise'>`. However, the latter is only valid for RPC calls and state queries, not signing stuff or creating extrinsics, I believe.

Until the SIWFv2 stuff went in, this wasn't really an issue... I'm not a fan of type-casting in most cases, but here it seems pretty inocuous.